test/pages/Inventory.page.js

```diff
@@ -9,9 +9,26 @@ class InventoryPage extends BaseSwagLabPage {
 
     get addItemToCartBtns() { return $$('[id^="add-to-cart"]'); }
 
+    get allInventoruItem() { return $$('//div[@class="inventory_list"]/child::*'); }
+
+    get sortlowToHigh(){return $('//option[@value="lohi"]' )}
+    get highToLow(){return $('//option[@value="hilo"]')}
+    get nameAtoZ(){return $('//option[@value="az"]' )}
+    get nameZtoA(){return $('//option[@value="za"]' )}
+
+    sortingDropDownList(i){return $(`//select[@class="product_sort_container"]/option[@value="${i}"]`)}
```

Comment:
I'd rename this method to smth like:
```suggestion
    changeSortingOrder(sortingOrder){return $(`//select[@class="product_sort_container"]/option[@value="${i}"]`)}
```